src/koga/ninja.lisp

```diff
@@ -4,6 +4,20 @@
   (declare (ignore configuration name))
   (ninja:make-line-wrapping-stream (ninja:make-timestamp-preserving-stream path)))
 
+(defun lisp-command (script &optional arguments)
+  (concatenate 'string
+               "$lisp "
+               #+abcl "--noinform --batch --noinit "
```

Comment:
If it is possible for the `:lisp` config argument to not be the implementation koga is running lisp-command on, the option flags ought to be based on the former rather than the latter. And if it is not possible there probably shouldn't be a `:lisp` config option.

Reply:
Right now it is just there in case the binary is not named something obvious. CCL does this sometimes with `ccl64`.